contrib/pr/checkout.go

```diff
@@ -247,7 +247,8 @@ func main() {
 			log.Fatalf("Failed to duplicate this code file in PR : %v", err)
 		}
 	}
-	time.Sleep(5 * time.Second)
+	//Force build of js, css, bin, ...
+	runCmd("make", "build")
```

Comment:
I'd consider running `clean-all` as well. I'm having problems when I'm careless about leftover files from other branches (e.g. temporary files that are marked as ignored remain in the tree, but in a different branch are considered added/untracked files).

Perhaps `clean-all` should be a dependency of `pr`?

Reply:
Should build just be a dependency as well, then, rather than add the call here?